proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -127,6 +155,10 @@ message QueryConsumerChainStopProposalsResponse {
 message Chain {
   string chain_id = 1;
   string client_id = 2;
+  // If the `chainID` is an Opt-In chain, i.e., no validator is forced to validate chain `chainID`
+  bool opt_In = 3;
```

Comment:
Why do we need this? (it's more clear for users is a perfectly fine reason if you think it applies)

 Can't we keep the TopN == 0 means it's an opt-in chain, and just have the TopN field?

Reply:
We don't need it in the logic. The intention was to make it clearer for the user. But after some thought, it can actually be removed.